### ISSUE TITLE
Extract instructors and workshops output for other countries

### ISSUE THREAD

**Issue by ctmartins** [softwaresaved/carpentry-data-analysis#14]
Solve the problem that when extracting other countries data it doesn't save in the corresponding folder (instructors and workshops). The excel files just stay in the data folder.

**Comment by ctmartins** [comment]
To solve this it seems like we just need to change line 54 and 56 of clp-parser.rb
Right now is like this:
```
54  options.workshops_file = File.join(DATA_DIR, "carpentry-workshops_#{country_code}_#{date}.csv")
...
56  options.instructors_file = File.join(DATA_DIR, "carpentry-instructors_#{country_code}_#{date}.csv")
```
We just need to substitute DATA_DIR for WORKSHOPS_DIR and INSTRUCTORS_DIR, respectively.

**Comment by anenadic** [comment]
Fixed via 1827def.
